perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/RegistrarManagerImpl.java

```diff
@@ -3689,6 +3734,46 @@ public void deleteGroupsFromAutoRegistration(PerunSession sess, List<Group> grou
 		perun.getGroupsManagerBl().deleteGroupsFromAutoRegistration(sess, groups);
 	}
 
+	@Override
+	public void deleteGroupsFromAutoRegistration(PerunSession sess, List<Group> groups, ApplicationFormItem formItem) throws GroupAlreadyRemovedException, GroupNotExistsException, PrivilegeException {
```

Comment:
Add the check that form item exists here ( I know its in RegistrarManagerMethod, but in case someone calls this method not through RPC)